docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -65,16 +65,20 @@ The rollout can be triggered by running the following command:
 clusterctl alpha rollout restart kubeadmcontrollplane/my-kcp
 ```
 
-To do the same for machines managed by a `MachineDeployment` it's enough to make an arbitrary
-change to its `Spec.Template`, one common approach is to run:
+Similarly, to do the same for machines managed by a `MachineDeployment` it has field `RolloutAfter` that 
```

Comment:
Q: Can we just have this entire test once for KCP & MD and then at the end we can have one command for KCP & MD each